animation_nodes/algorithms/perlin_noise.pyx

```diff
@@ -26,6 +27,54 @@ def wiggleVectorList(amount, double evolution, amplitude, int octaves, double pe
         values[i] = perlinNoise1D(evolution + i * 354623, persistance, octaves) * _amplitude[i % 3]
     return result
 
+def wiggleFloatList(amount, double evolution, amplitude, int octaves, double persistance):
```

Comment:
Float Lists in Animation Nodes are actually of type `DoubleList` not `FloatList`, so this function should be edited to reflect that.